.clang-format

```diff
@@ -0,0 +1,76 @@
+# Copyright (C) 2016 Olivier Goffart <ogoffart@woboq.com>
+#
+# You may use this file under the terms of the 3-clause BSD license.
+# See the file LICENSE from this package for details.
+
+# This is the clang-format configuration style to be used by Qt,
+# based on the rules from https://wiki.qt.io/Qt_Coding_Style and
+# https://wiki.qt.io/Coding_Conventions
+
+---
+# Webkit style was loosely based on the Qt style
+BasedOnStyle: WebKit
+
+Standard: c++14
```

Comment:
This was changed to c++17 in https://github.com/COVESA/vsomeip/commit/cf497232adf84f55947f7a24e1b64e04b49f1f38

Reply:
I will check with the team

Reply:
done in https://github.com/COVESA/vsomeip/pull/733/commits/d452c10902b95ffac25c651fe0bc8f2a68ef4515